README.md

```diff
@@ -39,11 +39,10 @@ var job = new CronJob(
     true,
     'America/Los_Angeles'
 );
-// Use this if the 4th param is default value(false)
-// job.start()
+// job.start() - See note below when to use this
 ``` 
 
-Note - You don't need to explicitly start a job in order to make it run since the 4th parameter is set to `true`. However, by default you need to call `job.start()` to start the cron job, which gives a little more control over running your jobs. 
+Note - By default, jobs are started automatically after constructing a `CronJob()` object. You can turn this feature off by setting the 4th `CronJob()` parameter to `false` so job(s) can be started when needed.
```

Comment:
@intcreator Oof, my apologies! I'm not sure how I messed up my interpretation this badly. Would this be better/acceptable?
```suggestion
Note - In the above example, the 4th parameter of `CronJob` automatically starts the job upon creation. By default, jobs need to be programmatically started using `CronJob.start()` if this parameter is not specified/falsy.
```